datafangst.md

```diff
@@ -2,6 +2,20 @@
 
 ## Leveranser 2020
 
+## Februar
+
+### Datafangst 2020-1.1.0
+* Vi har gjeninnført støtten for å hente data fra Les API v2.
+* Import av objektliste fra Excel-fil skal nå fungere for alle nye
```

Comment:
Jeg synes det er lurt med features / de viktigste sakene først. Tror denne betyr mest for brukere, så ville hatt den øverst.